doc/classes/CodeEdit.xml

```diff
@@ -548,28 +548,28 @@
 			Sets the string delimiters. All existing string delimiters will be removed.
 		</member>
 		<member name="gutters_draw_bookmarks" type="bool" setter="set_draw_bookmarks_gutter" getter="is_drawing_bookmarks_gutter" default="false">
-			Sets if bookmarked should be drawn in the gutter. This gutter is shared with breakpoints and executing lines.
+			If [code]true[/code], bookmarks are drawn in the gutter. This gutter is shared with breakpoints and executing lines. See [method set_line_as_bookmarked].
 		</member>
 		<member name="gutters_draw_breakpoints_gutter" type="bool" setter="set_draw_breakpoints_gutter" getter="is_drawing_breakpoints_gutter" default="false">
-			Sets if breakpoints should be drawn in the gutter. This gutter is shared with bookmarks and executing lines.
+			If [code]true[/code], breakpoints are drawn in the gutter. This gutter is shared with bookmarks and executing lines. Clicking the gutter will toggle the breakpoint for the line, see [method set_line_as_breakpoint].
 		</member>
 		<member name="gutters_draw_executing_lines" type="bool" setter="set_draw_executing_lines_gutter" getter="is_drawing_executing_lines_gutter" default="false">
-			Sets if executing lines should be marked in the gutter. This gutter is shared with breakpoints and bookmarks lines.
+			If [code]true[/code], executing lines are marked in the gutter. This gutter is shared with breakpoints and bookmarks. See [method set_line_as_executing].
 		</member>
 		<member name="gutters_draw_fold_gutter" type="bool" setter="set_draw_fold_gutter" getter="is_drawing_fold_gutter" default="false">
-			Sets if foldable lines icons should be drawn in the gutter.
+			If [code]true[/code], the fold gutter is drawn. In this gutter, the [theme_item can_fold_code_region] icon is drawn for each foldable line (see [method can_fold_line]) and the [theme_item folded_code_region] icon is drawn for each folded line (see [method is_line_folded]). These icons can be clicked to toggle the fold state, see [method toggle_foldable_line]. [member line_folding] must be [code]true[/code] to show icons.
 		</member>
 		<member name="gutters_draw_line_numbers" type="bool" setter="set_draw_line_numbers" getter="is_draw_line_numbers_enabled" default="false">
-			Sets if line numbers should be drawn in the gutter.
+			If [code]true[/code], the line number gutter is drawn. Line numbers start at [code]1[/code] and are incremented for each line of text. Clicking and dragging in the line number gutter will select entire lines of text.
 		</member>
 		<member name="gutters_zero_pad_line_numbers" type="bool" setter="set_line_numbers_zero_padded" getter="is_line_numbers_zero_padded" default="false">
-			Sets if line numbers drawn in the gutter are zero padded.
+			If [code]true[/code], line numbers drawn in the gutter are zero padded based on the highest number. [member gutters_draw_line_numbers] must be [code]true[/code].
```

Comment:
Maybe...? Up to your discretion.
```suggestion
			If [code]true[/code], line numbers drawn in the gutter are zero padded based on the total line count. [member gutters_draw_line_numbers] must be [code]true[/code].
```
Not a fan of "must be [code]true[/code]" because it doesn't really explain it, although sure, you can infer it.

Reply:
I could have the last sentence be "Line number gutters must be enabled, see [member gutters_draw_line_numbers]." instead.

Reply:
Actually on a second look the following is common enough in the class reference:
```suggestion
			If [code]true[/code], line numbers drawn in the gutter are zero padded based on the total line count. Requires [member gutters_draw_line_numbers] to be [code]true[/code].
```
or 
```suggestion
			If [code]true[/code], line numbers drawn in the gutter are zero padded based on the total line count. Requires [member gutters_draw_line_numbers] to be set to [code]true[/code].
```